react/features/invite/components/add-people-dialog/native/AddPeopleDialog.js

```diff
@@ -230,7 +230,6 @@ class AddPeopleDialog extends AbstractAddPeopleDialog<Props, State> {
                         data = { inviteItems }
                         horizontal = { true }
                         keyExtractor = { this._keyExtractor }
-                        keyboardShouldPersistTaps = 'always'
```

Comment:
On android the footer seems to adjust when the keyboard is shown, but on ios the footer is hidden when the same action takes place. In this case, I think we should offer the possibility for the keyboard to be dismissed.